### ISSUE TITLE
TV show always plays first few minutes and then pauses

### ISSUE THREAD

**Issue by krugster** [OstlerDev/PopcornTV#232]
I've noticed that (almost?) every tv show will start playing through it's initial buffer and then get to the end of that and pause.  I've found that if I slowly reverse and then pause, the rest of it starts loading almost immediately.

I don't see anything in the logs, so was hoping you've seen something similar?

**Comment by OstlerDev** [comment]
Yeah, I have seen this happen though I am not really sure why it does happen. It will sometimes stream 30 seconds then pause then resume shortly after. I do not know why this is happening and there are no logs as to why it is happening...

**Comment by krugster** [comment]
Another issue that seems very related is that sometimes a tv show will seem to have very good download speeds (the buffer will fill quickly with >.3MB) but then it will only allow me to play the first 30 seconds or so and never load the rest of the show.  From what I can tell the download has stopped and I am still seeing this error in the logs:
2015-09-22T02:09:55.995Z - error:  Error: EPERM, unlink 'C:\Users\pkrug\PopcornTV\assets\torrent-stream\41ab949e770b3fea249a4a30746804890b61015a\Modern.Family.S06E06.HDTV.x264-KILLERS.mp4'
    at Error (native)

**Comment by krugster** [comment]
Yeah, looking back through the logs that seems to be the consistent error when it won't get past the 30s.

**Comment by krugster** [comment]
And then after restarting the server and deleting all the underlying directories it worked on the same file the next time around.  It still paused in the very beginning but then it continued...not sure if that helps?
